site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/converter/SiteConverter.java

```diff
@@ -17,10 +21,27 @@ public Site convert(final Organization org) {
                 org.getName(),
                 (org.getAlias().isEmpty()) ? null : org.getAlias().get(0).getValueAsString(),
                 findParentSiteId(org),
-                org.getImplicitRules()
+                org.getImplicitRules(),
+                findAddress(org)
         );
     }
 
+    private Address findAddress(Organization org) {
+        if (org.hasAddress()) {
+            List<StringType> line = org.getAddress().get(0).getLine();
```

Comment:
A small thing, but "line" confused me since this is the a list of all the address lines

Reply:
Renamed to final List<StringType> lines = org.getLine();